src/darwin/Framework/CHIP/MTRDeviceController_XPC.mm

```diff
@@ -33,6 +35,8 @@
 @interface MTRDeviceController_XPC ()
 
 @property (nonatomic, retain, readwrite) NSUUID * uniqueIdentifier;
+@property (nonatomic, readonly, nullable) NSMapTable * nodeIDToDeviceMap;
```

Comment:
This should be exported from an internal header not re-declared here, otherwise they're a shadow

Reply:
Put these in MTRDeviceController_Internal.h